### PR TITLE
minor adjustments to enable building react-native-win32.dll with the …

### PR DESCRIPTION
**Pull request by aeulitz**
…Office compiler

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/3359)

### CODE REVIEW

File: vnext/IntegrationTests/React.Windows.IntegrationTests.vcxproj [hunk 1]
```diff
@@ -29,7 +29,6 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{700A84FD-F92A-43F1-8D06-B0E0745DF9B5}</ProjectGuid>
     <Keyword>StaticLibrary</Keyword>
-    <AppContainerApplication>true</AppContainerApplication>
```
Comment (JunielKatarn) [inline vnext/IntegrationTests/React.Windows.IntegrationTests.vcxproj]: Why is this change needed? (Not that I mind).

Comment (aeulitz) [reply vnext/IntegrationTests/React.Windows.IntegrationTests.vcxproj]: Without it, the office compiler ends up including vccorlib.h and then trips over the C++/CX language features in it, resulting in several errors like 

d:\iss\private-rnw\vnext\packages\visualcpptools.internaladdchpe.vs2017layout.14.13.26133.2\lib\native\include\vccorlib.h(934): error C2059: syntax error: ':' (compiling source file TestRootView.cpp) [D:\ISS\private-rnw\vnext\IntegrationTests\React.Windows.IntegrationTests.vcxproj]
